### PR TITLE
Add kava (testnet and mainnet) to setup_service and price_service

### PR DESCRIPTION
**Pull request by pkakelas**
### Make sure these boxes are checked! 📦✅

- [ 👍] You ran `./run_tests.sh`
- [👍 ] You ran `pre-commit run -a`
- [👍 ] If you want to add your network to `setup_service.py`, provide a link to your
    [safe-deployments PR](https://github.com/safe-global/safe-deployments/pulls) and check network name
    exists in [safe-eth-py](https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py)
    - https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L261
    - https://github.com/safe-global/safe-eth-py/blob/master/gnosis/eth/ethereum_network.py#L381
    - https://github.com/safe-global/safe-deployments/pull/145
    - https://github.com/safe-global/safe-deployments/pull/137

### CODE REVIEW

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 1]
```diff
@@ -362,6 +362,14 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 14062206, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 14062208, "1.3.0"),
     ],
+    EthereumNetwork.KAVA_EVM: [
```
Comment (moisses89) [inline safe_transaction_service/history/management/commands/setup_service.py:365]: `KAVA_EVM_MAINNET` is preferable to be aligned with the other chain names.

File: safe_transaction_service/history/management/commands/setup_service.py [hunk 2]
```diff
@@ -362,6 +362,14 @@ def create_task(self) -> Tuple[PeriodicTask, bool]:
...
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2116303, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2116307, "1.3.0"),
+    ],
+    EthereumNetwork.KAVA_EVM_TESTNET: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2241523, "1.3.0+L2"),
+        ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2241523, "1.3.0"),
```
Comment (moisses89) [inline safe_transaction_service/history/management/commands/setup_service.py:371]: In this block only have one transaction to create the proxyFactory. Then I couldn't confirm that this contracts were deployed at this block. 
https://explorer.testnet.kava.io/block/2241523/transactions#txs

**Review by moisses89 — CHANGES_REQUESTED** [review]

File: safe_transaction_service/history/management/commands/setup_service.py (hunk 1 above)
Comment (moisses89) [reply safe_transaction_service/history/management/commands/setup_service.py:365]: This is ok, it's the same name that is in safe-eth-py.

**Comment by Uxio0** [comment]
I'm closing this as it's inactive

**Pull request closed without merging**